server/sonar-server/src/main/java/org/sonar/server/qualitygate/QualityGates.java

```diff
@@ -114,25 +86,11 @@ public QualityGateDto copy(long sourceId, String destinationName) {
     }
   }
 
-  public Collection<QualityGateDto> list() {
-    try (DbSession dbSession = dbClient.openSession(false)) {
-      return dao.selectAll(dbSession);
-    }
-  }
-
-  public void delete(long idToDelete) {
-    checkIsQualityGateAdministrator();
-    QualityGateDto qGate = getNonNullQgate(idToDelete);
-    try (DbSession session = dbClient.openSession(false)) {
-      if (isDefault(qGate)) {
-        propertiesDao.deleteGlobalProperty(SONAR_QUALITYGATE_PROPERTY, session);
-      }
-      propertiesDao.deleteProjectProperties(SONAR_QUALITYGATE_PROPERTY, Long.toString(idToDelete), session);
-      dao.delete(qGate, session);
-      session.commit();
-    }
-  }
-
+  /**
+   * Use {@link QualityGateUpdater#setDefault(DbSession, QualityGateDto)}
+   * @deprecated
+   */
+  @Deprecated
   public void setDefault(DbSession dbSession, @Nullable Long idToUseAsDefault) {
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1133)